Resources/ServerInfo/Guidebook/Engineering/TEG.xml

```diff
@@ -1,18 +1,18 @@
-﻿<Document>
+<Document>
   # Thermo-electric Engine (TEG)
 
-  The TEG generates power by exchanging heat between hot and cold gases. On station, hot gas is usually created by burning plasma, and an array of heat-exchanging pipes in space radiates away heat to make a cold side.
+  The TEG generates power by exchanging heat between hot and cold gases. On the station, hot gas is usually created by burning plasma, and an array of heat-exchanging pipes in space radiates away heat cools down circulated gas.
```

Comment:
```suggestion
  The TEG generates power by exchanging heat between hot and cold gases. On the station, hot gas is usually created by burning plasma, and an array of heat-exchanging pipes in space radiates away heat to cool down circulated gas.
```

Reply:
sentence structure can still be improved after change, thanks for highlighting this skill issue ( i swear i spent an hour spellchecking this )